requirements.txt

```diff
@@ -1,2 +1,4 @@
 Flask
-waitress
\ No newline at end of file
+waitress
+thefuzz
+os
```

Comment:
Since the `os` library is built in to Python, the entry for it in `requirements.txt` should be removed. Having it in the file causes an error when you try to use it to install dependencies:

```
pip install -r requirements.txt
Collecting Flask (from -r requirements.txt (line 1))
  Downloading flask-3.0.3-py3-none-any.whl.metadata (3.2 kB)
Collecting waitress (from -r requirements.txt (line 2))
  Using cached waitress-3.0.0-py3-none-any.whl.metadata (4.2 kB)
Collecting thefuzz (from -r requirements.txt (line 3))
  Using cached thefuzz-0.22.1-py3-none-any.whl.metadata (3.9 kB)
ERROR: Could not find a version that satisfies the requirement os (from versions: none)
ERROR: No matching distribution found for os
```